firmware/application/apps/ui_settings.cpp

```diff
@@ -160,14 +160,26 @@ SetRadioView::SetRadioView(
         });
     }
 
-    add_children({&check_clkout,
-                  &field_clkout_freq,
-                  &labels_clkout_khz,
-                  &value_freq_step,
-                  &labels_bias,
-                  &check_bias,
-                  &button_save,
-                  &button_cancel});
+    if (hackrf_r9) {
+        add_children({&check_clkout,
+                      &field_clkout_freq,
+                      &labels_clkout_khz,
+                      &value_freq_step,
+                      &labels_bias,
+                      &check_bias,
+                      &force_tcxo,
```

Comment:
Nit: you can call add child separately with just the one extra control. If that’s the only difference between the two.

Reply:
Or right, I guess order doesn't matter does it